WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderUtils.java

```diff
@@ -331,6 +358,21 @@ public static ReaderTagList getOrderedTagsList(ReaderTagList tagList, Map<String
         return orderedTagList;
     }
 
+    public static boolean containsFollowing(ReaderTagList tagList) {
```

Comment:
Just for clarity, what do you think about renaming this on to `containsFollowingTag`? Also, we can even potentially move this method into `ReaderTagList` so it would be able to tell us by itself - does it contain the following tag or not :)

Reply:
Yep! Like it, changed [here](bb5c692fbbb87476105885a2bd7154885377a04f) 👍